docsets/Powershell/docset.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "Powershell",
-    "version": "/22-07-24",
+    "version": "/23-01-16",
     "archive": "Powershell.tgz",
```

Comment:
This archive should be at this path: `docsets/Powershell/Powershell.tgz`. It should not be in the root folder of the repo.